R/ConceptSet.R

```diff
@@ -33,11 +33,14 @@
 #' @export
 getConceptSetName <- function(baseUrl, setId, formatName = FALSE) {
   .checkBaseUrl(baseUrl)
-
-  url <- gsub("@baseUrl", baseUrl, gsub("@setId", setId, "@baseUrl/conceptset/@setId"))
+  
+  url <-
+    gsub("@baseUrl",
```

Comment:
sprintf is a cleaner function for this: 
`sprintf("%s/conceptset/%s", baseUrl, setId)`

Reply:
Maybe it is not part of this PR. It was already there?

I agree - we should use consistently. i like https://github.com/tidyverse/glue and find that to be better and easier to follow.

Reply:
My bad @gowthamrao -- didn't know this was already coded like this.